src/components/List/List.js

```diff
@@ -1,39 +1,34 @@
-import React from "react";
-import propTypes from "prop-types";
+import React from 'react';
+import propTypes from 'prop-types';
 
-import styled from "styled-components";
-import { createBorderStyles, createBoxStyles } from "../common";
+import styled from 'styled-components';
+import { createBorderStyles, createBoxStyles } from '../common';
 
-const createListPositionStyles = ({
-  verticalAlign = "bottom",
```

Comment:
and lets keep the default values in `createListPositionStyles` (it's made that way in order to make component position absolute only when user specifies either one of: `verticalAlign` and `horizontalAlign`). We'll change API for that component in V3

Reply:
sure, gonna add a temporary `undefined` value to those properties so we don't have to disable the ESlint rule for props without default values and we fix that on v3 :)